src/NetworkCanvas.js

```diff
@@ -241,6 +241,10 @@ export default class NetworkCanvas extends EventEmitter {
   setAnnotations = annotations => annotations.forEach(a => {
     const start = NetworkNode.findById(a.targets[0].id);
     const end = NetworkNode.findById(a.targets[1].id);
+    if (!start || !end) {
+      console.warn('Could not find start or end node for annotation', a);
+      return; 
+    }
     this.addEdge(new NetworkEdge(a.id, start, end, a.bodies));
```

Comment:
Likewise: is this just to be more robust against consistency problems? Or does this case happen in your app due to lazy loading of annotations?

Reply:
It can happen that the endpoints of a relation are not on screen and therefore were not retrieved from the backend.